transports/http/src/main/java/org/mule/transport/http/filters/HttpBasicAuthenticationFilter.java

```diff
@@ -135,6 +135,7 @@ public void authenticateInbound(MuleEvent event)
                 if (logger.isDebugEnabled())
                 {
                     logger.debug("Authentication request for user: " + username + " failed: " + e.toString());
+                    logger.debug("Cause: " + e.getCause().getMessage());
```

Comment:
Doesn't this work by logging the exception directly? To get the entire stack logged.